repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/content-cards.ftl

```diff
@@ -15,8 +18,12 @@
                 <#assign clickableCard=false/>
             </#if>
 
-            <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+            <li class="nhsuk-grid-column-one-${size} nhsuk-card-group__item">
                 <div class="nhsuk-card ${clickableCard?then('nhsuk-card--clickable', 'nhsuk-card')}">
+                    <@hst.link var="cardImage" hippobean=contentCard.cardImage />
+                    <#if cardImage??>
+                    <img class="nhsuk-card__img" src="${cardImage}" alt="">
```

Comment:
Could we render `cardImage` description as alt text ?

Reply:
done